app/views/layouts/_footer.haml

```diff
@@ -1,8 +1,24 @@
 %div.pull-left
   Done by #{Rails.application.config.application_creator}
+  %a{ :href => "https://fr.linkedin.com/pub/bastien-barbe/a4/61b/118",
+    :style => "text-decoration:none;", :target => "_blank" }
+    %span
+      %img{ :src => "https://static.licdn.com/scds/common/u/img/webpromo/btn_in_20x15.png",
+        :width => "20", :height => "15", :alt => "LinkedIn Bastien BARBE",
+        :style => "vertical-align:middle;", :border => "0" }
+  %br
+  %a{ :href => "https://twitter.com/share", :class => "twitter-share-button", "data-text" => "Check it out",
+    "data-via" => "BastienBARBE", "data-related" => "BastienBARBE",
+    "data-hashtags" => "developer", "data-url" => "http://www.bastienbarbé.com" }
+  %a{ :href => "https://www.hopwork.fr/profile/bastienbarbe", "data-id" => "56798cf6e4b0c3293a985904",
+    :class => "hopwork_button", "data-layout" => "scare", "data-colorscheme" => "blue",
+    "data-recos" => "true", "data-bgrecos" =>"dark" }
+  %iframe{ :src => "https://ghbtns.com/github-btn.html?user=BastienBABAR&type=follow&count=true",
+    "frameborder" => "0", "scrolling" => "0", :width => "180px", :height => "20px" }
 %div.pull-right
   %nav
     %ul.list-inline
       %li
         %span#cdSiteSeal1
-          %script{:src => "//tracedseals.starfieldtech.com/siteseal/get?scriptId=cdSiteSeal1&cdSealType=Seal1&sealId=55e4ye7y7mb730aa1fd9889fd9470d76zzxxy7mb7355e4ye74ffe7b91ae2b110", :type => "text/javascript"}
+          %script{:src => "//tracedseals.starfieldtech.com/siteseal/get?scriptId=cdSiteSeal1&cdSealType=Seal1&sealId=55e4ye7y7mb730aa1fd9889fd9470d76zzxxy7mb7355e4ye74ffe7b91ae2b110",
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>Line is too long. [183/80]